homeassistant/components/fing/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The Fing integration."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .coordinator import FingDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.DEVICE_TRACKER]
+
+type FingConfigEntry = ConfigEntry[FingDataUpdateCoordinator]
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: FingConfigEntry) -> bool:
+    """Set up the Fing component."""
+    config_entry.async_on_unload(
+        config_entry.add_update_listener(async_update_listener)
+    )
```

Comment:
why?

Reply:
I was keeping it for possible support for options, but for now I'll remove it.